index.html

```diff
@@ -7,10 +7,91 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+<!-- Main Content -->
+<main>
+<header>
+<div class="header">
+    <div class="upper">
+        <div class="container">
+            <img class="logo" src="/images/logo.png" alt="logo">
+        </div>
+        <div class="container-right">
+            <h1>Your Cake Mate</h1>
+            <h2>For Your Orders <br> 07403720162</h2>
+        </div>
+    </div>
+    <div>
+        <nav>  
+            <div class="topnav">
+              <a href="#">Home</a>
+              <a href="#">Cakes</a>
+              <a href="#">Ordering</a>
+              <a href="#">Lessons</a>
+              <a href="#">About</a>
+            </div>
+        </nav> 
+    </div>
+</header>
+
+
+
+    <div class="row">
+        <div class="rightcolumn">
```

Comment:
be careful with this kind of class name. On mobile, this is not on the right! Be a little more descriptive of what the section _does_ rather than what it looks like.